x-pack/plugins/snapshot_restore/public/app/sections/home/snapshot_list/snapshot_details/snapshot_details.tsx

```diff
@@ -0,0 +1,361 @@
+        id="xpack.snapshotRestore.snapshotDetails.itemfFailuresNoneLabel"
+        data-test-subj="srSnapshotDetailsFailuressNoneTitle"
+        defaultMessage="-"
+      />
+    );
+
+    content = (
+      <EuiDescriptionList textStyle="reverse">
+        <EuiFlexGroup>
+          <EuiFlexItem data-test-subj="srSnapshotDetailsVersionItem">
+            <EuiDescriptionListTitle>
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotDetails.itemVersionLabel"
+                data-test-subj="srSnapshotDetailsVersionTitle"
+                defaultMessage="Version / Version ID"
+              />
+            </EuiDescriptionListTitle>
+
+            <EuiDescriptionListDescription
+              className="eui-textBreakWord"
+              data-test-subj="srSnapshotDetailsVersionDescription"
+            >
+              {version} / {versionId}
+            </EuiDescriptionListDescription>
+          </EuiFlexItem>
+
+          <EuiFlexItem data-test-subj="srSnapshotDetailsIncludeGlobalUuidItem">
+            <EuiDescriptionListTitle>
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotDetails.itemUuidLabel"
+                data-test-subj="srSnapshotDetailsUuidTitle"
+                defaultMessage="UUID"
+              />
+            </EuiDescriptionListTitle>
+
+            <EuiDescriptionListDescription
+              className="eui-textBreakWord"
+              data-test-subj="srSnapshotDetailUuidDescription"
+            >
+              {uuid}
+            </EuiDescriptionListDescription>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiFlexGroup>
+          <EuiFlexItem data-test-subj="srSnapshotDetailsIncludeGlobalStateItem">
+            <EuiDescriptionListTitle>
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotDetails.itemIncludeGlobalStateLabel"
+                data-test-subj="srSnapshotDetailsIncludeGlobalStateTitle"
+                defaultMessage="Includes global state"
+              />
+            </EuiDescriptionListTitle>
+
+            <EuiDescriptionListDescription
+              className="eui-textBreakWord"
+              data-test-subj="srSnapshotDetailIncludeGlobalStateDescription"
+            >
+              {includeGlobalStateToHumanizedMap[includeGlobalState]}
+            </EuiDescriptionListDescription>
+          </EuiFlexItem>
+
+          <EuiFlexItem data-test-subj="srSnapshotDetailsStateItem">
+            <EuiDescriptionListTitle>
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotDetails.itemStateLabel"
+                data-test-subj="srSnapshotDetailsStateTitle"
+                defaultMessage="State"
+              />
+            </EuiDescriptionListTitle>
+
+            <EuiDescriptionListDescription
+              className="eui-textBreakWord"
+              data-test-subj="srSnapshotDetailStateDescription"
+            >
+              {state}
```

Comment:
would be nice to use `EuiHealth` for visual indication of these states

Reply:
I'll add it to the to-do list, since I need to get info from the ES team on what the different states are.